src/reanimated2/layoutReanimation/animationsManager.ts

```diff
@@ -95,7 +95,7 @@ function createLayoutAnimationManager() {
       if (!value) {
         return;
       }
-      stopObservingProgress(tag, value, true);
+      stopObservingProgress(tag, value);
```

Comment:
Previously there was `true` here passed, now we have default `false` - is that intended?

Reply:
Yes, this is intended. This parameter is related to Layout Aniamations. We don't want to clear Layout Animation configuration here.